var/spack/repos/builtin/packages/ginkgo/package.py

```diff
@@ -24,9 +24,9 @@ class Ginkgo(CMakePackage, CudaPackage, ROCmPackage):
 
     version("develop", branch="develop")
     version("master", branch="master")
-    version("1.6.0", commit="1f1ed46e724334626f016f105213c047e16bc1ae", preferred=True)  # v1.6.0
+    version("1.7.0", branch="49242ff89af1e695d7794f6d50ed9933024b66fe")  # v1.7.0
+    version("1.6.0", commit="1f1ed46e724334626f016f105213c047e16bc1ae")  # v1.6.0
     version("1.5.0", commit="234594c92b58e2384dfb43c2d08e7f43e2b58e7a")  # v1.5.0
-    version("1.5.0.glu_experimental", branch="glu_experimental")
```

Comment:
I guess this is needed by hiop - so cannot be removed now.

https://github.com/spack/spack/actions/runs/6828660917/job/18573266278?pr=40874

```
1. hiop: dependency on ginkgo@1.5.0.glu_experimental cannot be satisfied by known versions of ginkgo
```

cc: @cameronrutherford

Reply:
We are planning on supporting latest ginkgo with ExaSGD stack in future, but we do rely on that version at the moment...

Reply:
If you want to comment that line out in hiop/modify hiop to just depend on a `git.branch` version (if that's possible)  I think that would be ok...

Reply:
Right, I was anticipating that we would be able to update HiOp as well as all features are in develop and I was told it is easy, but since I don't think we can get that soon enough I will put this back. Thanks!

Reply:
cc @nychiang @pelesh

Reply:
> Right, I was anticipating that we would be able to update HiOp as well as all features are in 1.7.0 and I was told it is easy, but since I don't think we can get that soon enough I will put this back. Thanks!

@tcojeanis update should go in HiOp PR [669](https://github.com/LLNL/hiop/pull/669).